### PR TITLE
added exercise set 1

### PR DESCRIPTION
**Pull request by sroet**
This adds exercise set 1 (also talks a bit about markdown)

### CODE REVIEW

**Comment by sroet** [comment]
@andersle this one is ready for review and merge

File: exercises/01_Exercise_Set_1.ipynb [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Exercise 1.2** A chemist wants to inspect the flow of materials passing through a pipe in a factory. The\n",
+    "aim is to classify the flow into different types using acoustic sensors attached to the pipes. Before he can\n",
+    "use chemometric methods for this purpose he needs to convert the recordings into a data table.\n",
+    "1. Should each sound recording be represented as a **column** or a **row** vector in the data matrix?\n",
+    "2. What **variables** could be used here to describe the sound recordings?\n",
+    "3. Are there any problems related to the representation of the signals we need to be aware of when\n",
+    "analysing these acoustic samples?"
```
Comment (andersle) [inline exercises/01_Exercise_Set_1.ipynb]:
```suggestion
    "analyzing these acoustic samples?"
```

File: exercises/01_Exercise_Set_1.ipynb [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+   "metadata": {},
+   "source": [
+    "**Answer 1.2:** (double click here)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Exercise 1.3** A pharmaceutical company wants to create a new drug for malaria. In order to do this\n",
```
Comment (andersle) [inline exercises/01_Exercise_Set_1.ipynb]:
```suggestion
    "**Exercise 1.3** A pharmaceutical company wants to create a new drug for malaria. To do this,\n",
```

File: exercises/01_Exercise_Set_1.ipynb [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    "they use a *quantitative* structure-activity relationship (QSAR) approach. In QSAR a model is formed\n",
+    "(typically a regression model) which predicts the biological activity of a compound given information about\n",
+    "its molecular structure. To make a model, a set of compounds with known biological activity (here anti-\n",
```
Comment (andersle) [inline exercises/01_Exercise_Set_1.ipynb]:
```suggestion
    "its molecular structure. To make a model, a set of compounds with known biological activity (here: anti-\n",
```

File: exercises/01_Exercise_Set_1.ipynb [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    "malaria activity) and structure is used. The data matrix will have one row for each molecule and each\n",
+    "column contains a variable describing something related to the structure.\n",
+    "1. Can you suggest **variables** that may be used to describe the structure of the molecules?\n",
+    "2. Assume you decide to use XYZ coordinates for each atom as variables. What **problems** would you\n",
+    "encounter for such a representation?\n",
+    "3. Can you suggest ways to **solve** the problems occuring above?"
```
Comment (andersle) [inline exercises/01_Exercise_Set_1.ipynb]:
```suggestion
    "3. Can you suggest ways to **solve** the problems occurring above?"
```

File: exercises/01_Exercise_Set_1.ipynb [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Exercise 1.4** Assume you want to compare 20 spectra which all contain only three overlapping Gaussian\n",
+    "peaks. **Which** of the following situations can we expect to encounter problems when using the sampling\n",
+    "point representation (SPR)? SPR means that we sample the profile or shape at regular intervals where\n",
+    "each sample point is represented by a variable, see Figure 1 below:\n",
+    "1. Peak positions and widths are constant for all 20 spectra. Only the peak heights vary.\n",
+    "2. Peak positions are constant for all 20 spectra. Peak widths and heights vary.\n",
+    "3. Peak positions, widths and heights vary for the three peaks in the 20 spectra"
```
Comment (andersle) [inline exercises/01_Exercise_Set_1.ipynb]:
```suggestion
    "3. Peak positions, widths and heights vary for the three peaks in the 20 spectra."
```

File: exercises/01_Exercise_Set_1.ipynb [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
...gA2jKQ8yiqY8tmX06NEqX768OnTowHffShiGoaVLlya7zPvBguX9+/fVpUsX8/+G7Tt06FCKDbWYYWk7UitKr169WomJiWrRogUh2ZgrV66k+XhCQoK6dOmi27dva8GCBXJyciI02A0KlgBg45ydnTVw4EDt3r1bmzdvVrVq1bR7926CQaqcnJw0YMAARUZGavfu3apWrVqKl6Ii7zOZTPrqq690//599enTh0CsZJ2dOHFCAQEB+vnnn83Ly5Ytqzt37ujixYuSpJEjR+qvv/6Sp6cnl4TbCR8fH3Xr1k3nz5+3WJ5awTIqKorQrMjt27c1d+7cFB8bNWqUhg0bxmXANuiTTz5J9THDMMwNNVesWCFXV1cCg12hYAkAdoKmPMgomvLYBmdnZy1dulSRkZEaP348gViBXr166dixYwoMDNQ777yjmJgYmUwm1a5dWxERETp27JhCQ0MVExMjBwcHCpZ2wtfXVzdu3FBwcLDi4+PNy1O6JHzp0qUaOXIkoVmRbdu2afbs2cmWb9iwQXfu3FGbNm0Iycb8/vvvmjZtWoqPGYahvn376pdfftGKFSvk5uZGYLA7FCwBwM7QlAcZRVMe65cvXz6tXbtWn3/+ub799lsCyeMKFCigf/3rX3JwcNCMGTNUsmRJhYeHmy8L79atm/kycJPJxCXhdsLHx0cODg46efKk3n//fUl/N+L4448/VLx4cUnSvXv39Oabb+rll1/m8mErs2HDBu3atcs8izrJ6NGjNXToUO5baIO6d++e6j2IBw8erH379ik8PFxeXl6EBbtkMgzDIAYAsE/bt29Xr169VLp0aU2dOlUlSpQgFKRp//796tmzp7y9vTVt2jQ9/fTThGJFjh49qkaNGmn+/Plq2rQpgeRhZ86cUcWKFc2DWQ8PDwUGBurMmTO6efOmbt++LenvWXdr165VnTp1CM0OuLq6KjY2Vh4eHlq6dKn8/f3VsmVLnTp1SidPnlRQUJDOnj0rJycnXb9+Xc7OzoRmJcqUKaNz585p7Nixeu+99yT9Peuye/fu+vXXX7l3oY1ZsWKFunbtqujoaD1Ykhk2bJhWrVqlrVu3qkCBAoQFu8XPNABgx2jKg4yiKY91q1Chgr777jt16tTJ4v6IyHtKlSqlRo0amWdVxcTEKDIyUhcvXjQXK5Mww9J+eHh4mLeHTp06KTIyUmXK
```
Comment (andersle) [inline exercises/01_Exercise_Set_1.ipynb]:
```suggestion
    "that shifts of peaks can cause serious problems when analyzing the data. However, it is also observed that\n",
```

Comment (andersle) [inline exercises/01_Exercise_Set_1.ipynb]:
```suggestion
    "them to analyze the freshness of fruits and vegetables when they are out buying food. The constraints for\n",
```

**Comment by sroet** [comment]
@andersle this one has to wait untill #6 is merged (as I want to hyperref exercise 0 in the last cell)

**Comment by sroet** [comment]
@andersle hyperlink inserted, please have another look

**Comment by andersle** [comment]
@sroet: hyperlink works :-) that's a nice touch.

**Pull request merged**